library/src/blas1/rocblas_nrm2.cpp

```diff
@@ -49,7 +49,7 @@ namespace
         if(!x || !result)
             return rocblas_status_invalid_pointer;
 
-        size_t dev_bytes = rocblas_reduction_kernel_workspace_size<NB>(n, 1, result);
+        size_t dev_bytes = rocblas_reduction_kernel_workspace_size<NB, To>(n, 1);
```

Comment:
`rocblas_reduction_kernel_workspace_size` should have a default second argument of `1`, for when calculating non-batched.

Reply:
ok, done.